### PR TITLE
Public template

### PR DESCRIPTION
**Pull request by CEBergin**
Added test jenkinsfile.

### CODE REVIEW

File: Jenkinsfile-test [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+              }
+            }
+          }
+        }
+
+        
+      }
+    }
+
+    stage('Deploy to prod'){
```
Comment (mtCarto) [inline Jenkinsfile-test:156]: update stage name

File: Jenkinsfile-test [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+              ROCKET_DEPLOY_WEBHOOK
+            )
+            currentBuild.result = "FAILURE"
+            throw new Exception("Deploy failed")
+          }
+        }
+      }
+    }
+
+    // stage('BDD Tests') {
```
Comment (dhlevi) [inline Jenkinsfile-test:191]: Get rid of this section if it's not needed

File: Jenkinsfile-test [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        script {
+          try {
+            echo "Backing up..."
+            openshiftTag destStream: 'eagle-public', verbose: 'false', destTag: 'backup-pipeline-test', srcStream: 'eagle-public', srcTag: "pipeline-test"
+            sleep 5
+            echo "Deploying to prod..."
+            openshiftTag destStream: 'eagle-public', verbose: 'false', destTag: 'pipeline-test', srcStream: 'eagle-public', srcTag: "${IMAGE_HASH}"
+            sleep 5
+            // todo update namespace before switching over
+            openshiftVerifyDeployment depCfg: 'eagle-public', namespace: 'esm-dev', replicaCount: 1, verbose: 'false', verifyReplicaCount: 'false', waitTime: 600000
```
Comment (mtCarto) [inline Jenkinsfile-test:168]: switch to esm-test

File: src/app/projects/project-list/project-list.component.ts [hunk 4]
```diff
@@ -545,9 +545,8 @@ export class ProjectListComponent implements OnInit, OnDestroy {
       .takeUntil(this.ngUnsubscribe)
       .subscribe((res: any) => {
         if (res && res[0].data) {
-          this.tableParams.totalListItems = res[0].data.searchResults.length > 0
-                                          ? res[0].data.meta[0].searchResultsTotal
-                                          : 0;
+          this.tableParams.totalListItems =
```
Comment (dhlevi) [inline src/app/projects/project-list/project-list.component.ts]: Looks like you need to re-merge from develop!

**Pull request merged**